transports/quic/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "libp2p-quic"
+version = "0.8.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+description = "TLS based QUIC transport implementation for libp2p"
+repository = "https://github.com/libp2p/rust-libp2p"
+license = "MIT"
+
+[dependencies]
+if-watch = "2.0.0"
+libp2p-core = { version = "0.37.0", path = "../../core" }
+libp2p-tls = { version = "0.1.0-alpha", path = "../../transports/tls" }
+quinn = { version = "0.9.0", git = "https://github.com/quinn-rs/quinn.git", branch = "main", features = ["tls-rustls", "futures-io", "runtime-async-std"] }
```

Comment:
Does `quinn` support activating multiple runtime features? We'd need with our `tokio` and `async-std` features.

Reply:
> Does quinn support activating multiple runtime features?

Yep, you can choose what runtime you want to use for an `Endpoint` dynamically when you create it. You can even define new runtimes downstream!

> At the moment their crate at crates.io supports only tokio.

This is no longer true; 0.9 has been released.

Reply:
> I don't know what means "multiple". Two different runtimes simultaneously?

Not simultaneously in code but activated at the same time and choosing in the source code, which one you want.

> > Does quinn support activating multiple runtime features?
> 
> Yep, you can choose what runtime you want to use for an `Endpoint` dynamically when you create it. You can even define new runtimes downstream!

Nice, thank you!